.github/workflows/triage.yml

```diff
@@ -45,14 +39,23 @@ jobs:
         with:
           project-name: Support
           # If the author comes from Bitnami, send it to Bitnami. Otherwise, all to Triage
-          column-name: ${{ (contains(fromJson(env.BITNAMI_TEAM), github.actor)) && 'From Bitnami' || 'Triage' }}
+          # NOTE: This step has to be modified when the new column for automated PR is added
+          column-name: ${{ (github.actor == 'bitnami-bot' || (!contains(fromJson(env.BITNAMI_TEAM), github.actor))) && 'Triage' || 'From Bitnami' }}
           token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
           issue-number: ${{ github.event_name != 'issues' && github.event.number || github.event.issue.number }}
       - name: Triage labeling
         # Only if moved into Solved
         uses: andymckay/labeler@1.0.4
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
-          add-labels: ${{ (!contains(fromJson(env.BITNAMI_TEAM), github.actor)) && 'triage' || 'bitnami' }}
+          add-labels: ${{ github.actor == 'bitnami-bot' && 'auto-merge' || ((!contains(fromJson(env.BITNAMI_TEAM), github.actor)) && 'triage' || 'bitnami') }}
           # For reopened issues
-          remove-labels: "solved"
\ No newline at end of file
+          remove-labels: "solved"
+      - name: Verify labeling
+        # Only if the actor is bitnami-bot
+        if: ${{ github.actor == 'bitnami-bot' }}
```

Comment:
Will we trigger always this although is an issue? I'd add to the if condition that should be a PR (thinking in the future, if an action fails for example, we decide `bitnami-bot`to open an issue, then this step will be triggered).

Reply:
This code was removed, but you right I am not distinguishing between PRs and issues